.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+
+{
+	"name": "Codespaces for Thanos",
+
+	"context": "..",
+
+	"dockerFile": "",
+	
+	
+
+	"customizations": 
+	{
+		"vscode": {
+					"settings": { 
+									"go.toolsManagement.checkForUpdates": "local",
+									"go.useLanguageServer": true,
+									"go.gopath": "/go"
+			
+								}
+				}
+			
+			
+	},
+
+	"extensions": 
+	[
+		"ms-azuretools.vscode-docker",
+		"ms-kubernetes-tools.vscode-kubernetes-tools",
+		"davidanson.vscode-markdownlint",
+		"timonwong.shellcheck",
+		"docsmsft.docs-yaml",
+		"heptio.jsonnet",
```

Comment:
This is not the correct jsonnet extension. According to the VSCode Extension Marketplace, this is deprecated and no longer maintained. We should use the one made by Grafana (`grafana.vscode-jsonnet`), as I mentioned in the issue.

Reply:
Ops! How can I forget that

Reply:
Added in the latest commit. Thank You! :)